tests/fixtures/app/no-ember-data/config/ember-cli-update.json

```diff
@@ -0,0 +1,21 @@
+{
+  "schemaVersion": "1.0.0",
+  "packages": [
+    {
+      "name": "ember-cli",
+      "version": "<%= emberCLIVersion %>",
+      "blueprints": [
+        {
+          "name": "app",
+          "outputRepo": "https://github.com/ember-cli/ember-new-output",
+          "codemodsSource": "ember-app-codemods-manifest@1",
+          "isBaseBlueprint": true,
+          "options": [
+            "--no-ember-data",
```

Comment:
Hmm, it seems we don't add this option to the generated ember-cli-update.json file yet. I think we should, right?